test/CMakeLists.txt

```diff
@@ -650,10 +650,18 @@ if(${MIOPEN_TEST_WITH_MIOPENDRIVER})
         # Issue #1697: this is large test which has to run in serial and not enabled on gfx900/gfx906
         COMMAND MIOPEN_DRIVER_USE_GPU_REFERENCE=1 $<TARGET_FILE:MIOpenDriver> ${MIOPENDRIVER_MODE_CONV} -W 5078 -H 4903 -c 24 -n 5 -k 1 --fil_w 3 --fil_h 3 --pad_w 6 --pad_h 4 -F 1
     )
+
```

Comment:
[Recommendation][Quality] It would be better to add this testcase to the existing `test_regression_half_mi200`.
- Pros:
  - We do not like to use MIOpenDriver in tests and do it only when there is no other way.
- Cons:
  - You need to convert MIOpenDriver parameters to test_conv3d parameters, and check that test_conv3d fails without #1935 and passes with it. Which means some additional work.

Up to you.